File: internal/sast/fixtures/python/xss_negative.py
# internal/sast/fixtures/python/xss_negative.py
from flask import Flask, request, render_template_string

app = Flask(__name__)

@app.route("/echo")
def echo():
    msg = request.args.get("msg", "")
    return render_template_string("<p>{{ msg }}</p>", msg=msg)

File: internal/sast/fixtures/csharp/Xss_negative.cs
// internal/sast/fixtures/csharp/Xss_negative.cs
using Microsoft.AspNetCore.Mvc;

public class XssNegativeController : Controller
{
    public IActionResult Echo(string msg)
    {
        ViewBag.Msg = msg;
        return View();
    }
}
